src/components/routerView.vue

```diff
@@ -44,14 +45,18 @@
       return rejection.value.component
     }
 
-    const match = route.matches.at(depth)
+    const match: Route['matched'] | undefined = route.matches.at(depth)
```

Comment:
This type isn't totally necessary but since there's no registered router by default `route.matches` ends up being `any[]`. So `match` is also `any`. Adding this type inline gives us better type hints and checking.